File: 1332-remove-palindromic-subsequences/1332-remove-palindromic-subsequences.java
class Solution {
    public int removePalindromeSub(String s) {
        int N = s.length();
        for (int i = 0; i < N / 2; i++) {
            if (s.charAt(i) != s.charAt(N - 1 - i)) return 2;
        }
        return N > 0 ? 1 : 0;
    }
}

File: 1332-remove-palindromic-subsequences/1332-remove-palindromic-subsequences.cpp
class Solution {
public:
    int removePalindromeSub(string s) {
        int N = s.size();
        for (int i = 0; i < N / 2; i++) {
            if (s[i] != s[N - 1 - i]) return 2;
        }
        return N > 0;
    }
};